packages/hooks/src/__tests__/useCheck10kLimit.spec.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import type { TStores } from '@deriv/stores';
+import { renderHook } from '@testing-library/react-hooks';
+import useCheck10kLimit from '../useCheck10kLimit';
+import useMaxWithdrawAmount from '../useMaxWithdrawAmount';
+
+jest.mock('../useMaxWithdrawAmount');
+
+describe('useCheck10kLimit', () => {
+    test('should be false if useMaxWithdrawAmount is undefined', async () => {
+        (useMaxWithdrawAmount as jest.Mock).mockReturnValue('undefined');
+        const wrapper = ({ children }: { children: JSX.Element }) => (
+            <StoreProvider store={mockStore({}) as TStores}>{children}</StoreProvider>
```

Comment:
```suggestion
            <StoreProvider store={mockStore({})}>{children}</StoreProvider>
```
@hamid-deriv Same for other test files.

Reply:
Sorted.